src/components/DiffView/index.tsx

```diff
@@ -480,7 +481,7 @@ export class DiffViewBase extends React.Component<Props> {
         {extraMessages.length ? this.renderExtraMessages(extraMessages) : null}
 
         {diff && hunks && (
-          <React.Fragment key={`${diff.oldRevision}-${diff.newRevision}`}>
+          <React.Fragment key={version.id}>
```

Comment:
Is this going to work when, on a compare page, we select a different base or head version? Does it update the page correctly?

Reply:
Hmm, let me check that...

Reply:
It seems to work fine on dev. Changing either pushes a new route, which reloads most of the page, so yes, it works. I don't think there's a use case for just updating the `DiffView` in place for a different set of versions. The `diff` changes, which means that the `FileTree` and the `CodeOverview` also change.